internal/olm/operator/operator.go

```diff
@@ -78,6 +79,8 @@ type OLMCmd struct { // nolint:golint
 	Timeout time.Duration
 	// ForceRegistry forces deletion of registry resources.
 	ForceRegistry bool
+	// OLMNamespace is the namespace in which OLM is installed. Defaults to "olm".
```

Comment:
Only true if `AddToFlagSet` is used.
```suggestion
	// OLMNamespace is the namespace in which OLM is installed.
```

Reply:
solved.